components/scream/src/mct_coupling/scream_scorpio_interface_iso_c.F90

```diff
@@ -73,6 +108,35 @@ subroutine pio_update_time_c(filename_in,time) bind(c)
     call eam_update_time(trim(filename),time)
 
   end subroutine pio_update_time_c
+!=====================================================================!
+  subroutine get_variable_c(filename_in, shortname_in, longname_in, numdims, var_dimensions_in, dtype, pio_decomp_tag_in) bind(c)
```

Comment:
Are these functions here to stay, or will they disappear in a couple of code iterations? I think it's the latter, since we might not go to F90 anymore soon. But if they are here to stay (at least for some time), then I would consider adding a small comment at the top of the function, explaining in one line what the fcn does, as well as clarifying the meaning of any input that is not obvious. E.g., filename or var_dimensions is probably clear, but what's dtype? or pio_decomp_tag_in? They might be clear for you who have been working on pio for some time, but for others they might be obscure.

Reply:
Yep, I've added comments to every subroutine.  See Caldwell comment above.